README.md

```diff
@@ -72,15 +197,7 @@ base 10\*\*18).
 For a division example, 18/10 would be 18\*10\*\*18 \* 10\*\*18 /
 (10\*10\*\*18).
 
-## General information about Serpent
-
-- [The Serpent wiki](https://github.com/ethereum/wiki/wiki/Serpent)
-- [The Serpent guide](https://www.cs.umd.edu/~elaine/smartcontract/guide.pdf) (PDF)
-
-## General information about Augur
-
-- [A Roadmap For Augur and What’s Next](https://medium.com/@AugurProject/a-roadmap-for-augur-and-whats-next-930fe6c7f75a)
-- [Augur Master Plan](https://medium.com/@AugurProject/augur-master-plan-42dda65a3e3d)
+Also note, Mist uses a different contract abi style than usual, to convert use the following regex \((.\*?)\) and replace with nothing [except don't do it for events].
```

Comment:
I'd leave it back in tbh, I wasted hours once figuring this out, it's still a non-fixed bug in mist, etc. Although mist still doesn't even support contract bytecode uploads (it got broke about a year ago) so perhaps it's not too important